spiralfilm/core.py

```
@@ -538,3 +551,8 @@
                     num_tokens += tokens_per_name
         num_tokens += 3  # every reply is primed with <|start|>assistant<|message|>
         return num_tokens
+    def check_token_limit(self, messages):
```

Ruff (0.8.2)

Redefinition of unused `check_token_limit` from line 79

(F811)